src/components/Heatmap/Heatmap.jsx

```diff
@@ -11,13 +11,16 @@ import DotaMap from '../DotaMap';
  * Returns the adjusted heatmap data.
  */
 function scaleAndExtrema(points, scalef, max, shift) {
+  const averageValue = (points.reduce((a, b) => a + b.value, 0) / points.length);
+  const maxDeviationFromAvg = 25;
+
   const newPoints = points.map(p => ({
     x: Math.floor(p.x * scalef),
     y: Math.floor(p.y * scalef),
-    value: p.value + (shift || 0),
+    value: Math.min(p.value, (averageValue * maxDeviationFromAvg)) + (shift || 0),
```

Comment:
I would just compute averageValue * maxDeviationFromAvg once and reuse that variable (maybe call it maxValue)